app/javascript/common/formUtilities/validation.js

```diff
@@ -0,0 +1,23 @@
+export const required = value => (value ? undefined : 'Required');
+
+export const maxLength = max => value =>
+  value && value.length > max ? `Must be ${max} characters or less` : undefined;
+
+export const minLength = min => value =>
+  value && value.length < min ? `Must be ${min} characters or more` : undefined;
+
+export const number = value =>
+  value && Number.isNaN(Number(value)) ? 'Must be a number' : undefined;
+
+export const minValue = min => value =>
+  value && value < min ? `Must be at least ${min}` : undefined;
+
+export const email = value =>
+  value && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)
+    ? 'Invalid email address'
+    : undefined;
+
+export const alphaNumeric = value =>
+  value && /[^a-zA-Z0-9 ]/i.test(value)
```

Comment:
Maybe consider using the redux form validator npm package?

Reply:
Will look into using redux form validator npm package. Thanks.

Reply:
https://github.com/priley86/miq_v2v_ui_plugin/pull/42#discussion_r167996209